.gitignore

```diff
@@ -0,0 +1,23 @@
+# Compiled class file
```

Comment:
.gitignore file is not complete. Please try to generate new version (e.g. you can do it here https://www.toptal.com/developers/gitignore)
For example, it is missing to ignore files specific to the IDE you are using.